input/postgres/backends.go

```diff
@@ -5,23 +5,38 @@ import (
 	"fmt"
 	"strings"
 
+	"github.com/lib/pq"
 	"github.com/pganalyze/collector/state"
 	"github.com/pganalyze/collector/util"
 )
 
 const activitySQLDefaultOptionalFields = "waiting, NULL, NULL, NULL, NULL, NULL"
 const activitySQLpg94OptionalFields = "waiting, backend_xid, backend_xmin, NULL, NULL, NULL"
-const activitySQLpg96OptionalFields = "COALESCE(wait_event_type, '') = 'Lock', backend_xid, backend_xmin, wait_event_type, wait_event, NULL"
-const activitySQLpg10OptionalFields = "COALESCE(wait_event_type, '') = 'Lock', backend_xid, backend_xmin, wait_event_type, wait_event, backend_type"
-
-const activitySQL string = `SELECT (extract(epoch from COALESCE(backend_start, pg_catalog.pg_postmaster_start_time()))::int::text || pg_catalog.to_char(pid, 'FM0000000'))::bigint,
-				datid, datname, usesysid, usename, pid, application_name, client_addr::text, client_port,
-				backend_start, xact_start, query_start, state_change, %s, state, query
-	 FROM %s
-	WHERE pid IS NOT NULL`
+const activitySQLpg96OptionalFields = `COALESCE(wait_event_type, '') = 'Lock' as waiting, backend_xid, backend_xmin, wait_event_type, wait_event, NULL`
+const activitySQLpg10OptionalFields = `COALESCE(wait_event_type, '') = 'Lock' as waiting, backend_xid, backend_xmin, wait_event_type, wait_event, backend_type`
+
+const pgBlockingPidsField = `
+CASE
+	WHEN COALESCE(wait_event_type, '') = 'Lock' THEN pg_blocking_pids(pid)
+	ELSE NULL
```

Comment:
```suggestion
```
`NULL` is the default value when no `WHEN` clauses of a `CASE` statement are matched, so you could just omit this line.